test/conformance/kuttl/filter/exclude/user/wildcard/README.md

```diff
@@ -0,0 +1,12 @@
+## Description
+
+This test creates a policy, excluding users with wildcard `?*`.
+This policy denies pod creation.
+
+## Expected Behavior
+
+The pod should be accepted (user is `kubernetes-admin`).
```

Comment:
What about a test for a ServiceAccount and not user to ensure it's blocked?

Reply:
Done.